erpnext_retail/public/js/point_of_sale.js

```diff
@@ -1,28 +1,222 @@
+/* global Payment, __, frappe */
+
 class PointOfSaleNew extends erpnext.pos.PointOfSale{
-	make() {
-		return frappe.run_serially([
-			() => frappe.dom.freeze(),
-			() => {
-				this.prepare_dom();
-				this.prepare_menu();
-				this.set_online_status();
-			},
-			() => this.make_new_invoice(),
-			() => {
-				if(!this.frm.doc.company) {
-					this.setup_company()
-						.then((company) => {
-							this.frm.doc.company = company;
-							this.get_pos_profile();
-						});
-				}
-			},
-			() => {
-				frappe.dom.unfreeze();
-			},
-			() => this.page.set_title(__(this.frm.doc.company + ' Point of Sale'))
-		]);
-	}
+    make() {
+        return frappe.run_serially([
+            () => frappe.dom.freeze(),
+            () => {
+                this.prepare_dom();
+                this.prepare_menu();
+                this.set_online_status();
+            },
+            () => this.make_new_invoice(),
+            () => {
+                if(!this.frm.doc.company) {
+                    this.setup_company()
+                        .then((company) => {
+                            this.frm.doc.company = company;
+                            this.get_pos_profile();
+                        });
+                }
+            },
+            () => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 3 tabs but found 12 spaces.](https://app.codacy.com/gh/techlift-tech/erpnext_retail/pullRequest?prid=4838702)